src/cluster/purging.rst

```diff
@@ -129,45 +131,52 @@ consistency within the main database.
 
 Config Settings
 ====================================
-These settings ca be updated in the default.ini or local.ini:
+These settings can be updated in the default.ini or local.ini:
+
++-----------------------+--------------------------------------------+----------+
+| Field                 | Description                                | Default  |
++=======================+============================================+==========+
+| max_document_id_number| Allowed maximum number of documents in one | 100      |
+|                       | purge request                              |          |
++-----------------------+--------------------------------------------+----------+
+| max_revisions_number  | Allowed maximum number of accumulated      | 1000     |
+|                       | revisions in one purge request             |          |
++-----------------------+--------------------------------------------+----------+
+| allowed_purge_seq_lag | Beside purged_infos_limit, allowed         | 100      |
+|                       | additional buffer to store purge requests  |          |
++-----------------------+--------------------------------------------+----------+
+| index_lag_warn_seconds| Allowed durations when index is not        |          |
+|                       | updated for local purge checkpoint document| 86400    |
++-----------------------+--------------------------------------------+----------+
+
+During a database compaction,  we check all checkpoint purge docs. A client (an
+index or internal replication job) is allowed to have the last reported
+``purge_seq`` to be smaller than the current database shard's ``purge_seq`` by
+the value of ``(purged_infos_limit + allowed_purge_seq_lag)``.  If the client's
+``purge_seq`` is even smaller, and the client has not checkpointed within
+``index_lag_warn_seconds``, it prevents compaction of purge trees and we have to
+issue the following log warning for this client:
+
+.. code-block:: text
+
+    Purge checkpoint '_local/purge-mrview-9152d15c12011288629bcffba7693fd4’
+    not updated in 86400 seconds in
+    <<"shards/00000000-1fffffff/testdb12.1491979089">>
+
+If this type of log warnings occurs, check the client to see why the processing
```

Comment:
Should be "types of log warnings" or "type of log warning" - plurality agreement :)